src/os/vxworks/inc/vxworks7/os-impl-tasks.h

```diff
@@ -0,0 +1,51 @@
+/*
+ *  NASA Docket No. GSC-18,370-1, and identified as "Operating System Abstraction Layer"
+ *
+ *  Copyright (c) 2019 United States Government as represented by
+ *  the Administrator of the National Aeronautics and Space Administration.
+ *  All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * \file     os-impl-tasks.h
+ * \ingroup  vxworks
+ * \author   joseph.p.hickey@nasa.gov
+ *
+ */
+
+#ifndef INCLUDE_OS_IMPL_TASKS_H_
+#define INCLUDE_OS_IMPL_TASKS_H_
+
+#include <osconfig.h>
+#include <taskLib.h>
+
+
+/*tasks */
+typedef struct
+{
+    /* SDS: WIND_TCB was moved to a private taskLib file. There is an API to access fields within. In this case,
+     * OSAL wants to store other data with WIND_TCB in a single allocaiton of memory, so we only need the size. */
+    char tcb[VX_WIND_TCB_SIZE]; /* Must be first */
```

Comment:
Rather than forking this file - does vxworks 6.9 also have this `VX_WIND_TCB_SIZE` define?  Also do they give any alignment requirements for it?  My only concern with changing a real type to a generic `char[]` type is that you lose alignment.  As the parent struct contains `void*` and `long` it is probably aligned but worth noting.